.github/actions/setup/action.yml

```diff
@@ -5,13 +5,11 @@ inputs:
   setup-mops:
     description: Whether to setup mops
     required: false
-    default: true
-    type: boolean
```

Comment:
So the `boolean` type is the culprit? Is this documented somewhere?

Reply:
I don't know. It is just try and error 🤷‍♂️ I assume now that every input must be of string type

Reply:
https://github.com/actions/runner/issues/1483